.github/workflows/e2e.yml

```
@@ -301,16 +301,16 @@ jobs:
         working-directory: ./
     runs-on: self-hosted
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v4
       - name: Download artifact
         uses: actions/download-artifact@v3
```

actionlint (1.7.4)

the runner of "actions/download-artifact@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)